### PR TITLE
roave-backward-compatibility-check: add page

### PR DESCRIPTION
**Pull request by pxgamer**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/roave-backward-compatibility-check.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# roave-backward-compatibility-check
+
+> A tool that can be used to verify BC breaks between two versions of a PHP library.
```
Comment (schneiderl) [inline pages/common/roave-backward-compatibility-check.md]: I would prefer to have "backward compatibility breaks" istead of "BC breaks" as it looks easier to understand at first sight.

Comment (schneiderl) [inline pages/common/roave-backward-compatibility-check.md]: I would prefer to have "backward compatibility breaks" istead of "BC breaks" as it looks easier to understand at first sight.

Comment (pxgamer) [reply pages/common/roave-backward-compatibility-check.md]: Yeah, I had thought about it but the line was getting pretty long. Makes more sense for newcomers though. 👍

File: pages/common/roave-backward-compatibility-check.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+`roave-backward-compatibility-check --from={{git_reference}}`
+
+- Check for breaking changes between the last tag and a specific reference:
+
+`roave-backward-compatibility-check --to={{git_reference}}`
+
+- Check for breaking changes and output to markdown:
+
+`roave-backward-compatibility-check --format=markdown > results.md`
```
Comment (mebeim) [inline pages/common/roave-backward-compatibility-check.md]:
```suggestion
`roave-backward-compatibility-check --format=markdown > {{results.md}}`
```

**Pull request merged**